src/client/java/teammates/client/scripts/DataMigrationForSanitizedDataInAdminEmailAttributes.java

```diff
@@ -0,0 +1,102 @@
+package teammates.client.scripts;
+
+import teammates.client.remoteapi.RemoteApiClient;
+import teammates.client.scripts.helper.DataMigrationForSanitizedDataHelper;
+import teammates.client.scripts.helper.LoopHelper;
+import teammates.common.datatransfer.attributes.AdminEmailAttributes;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.SanitizationHelper;
+import teammates.logic.core.AdminEmailsLogic;
+import teammates.storage.api.AdminEmailsDb;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.google.appengine.api.datastore.Text;
+
+/**
+ * Script to desanitize content of AdminEmailAttributes if it is sanitized.
+ * Html sanitization of content before saving is removed and content is expected to be in its unsanitized form.
+ * This script helps to desanitize content of exisiting AdminEmailAttributes so that they follow the new standard.
+ */
+public class DataMigrationForSanitizedDataInAdminEmailAttributes extends RemoteApiClient {
+    private static final boolean isPreview = true;
+    private AdminEmailsDb adminEmailsDb = new AdminEmailsDb();
+    private AdminEmailsLogic adminEmailsLogic = AdminEmailsLogic.inst();
+    private int numberOfAffectedEmails;
+    private int numberOfUpdatedEmails;
+
+    public static void main(String[] args) throws IOException {
+        new DataMigrationForSanitizedDataInAdminEmailAttributes().doOperationRemotely();
+    }
+
+    @Override
+    protected void doOperation() {
+        List<AdminEmailAttributes> allEmails = adminEmailsLogic.getAllAdminEmails();
+        numberOfAffectedEmails = 0;
+        numberOfUpdatedEmails = 0;
+        LoopHelper loopHelper = new LoopHelper(100, "admin emails processed.");
+        println("Running data migration for sanitization on admin emails...");
+        println("Preview: " + isPreview);
+        for (AdminEmailAttributes email : allEmails) {
+            loopHelper.recordLoop();
+            fixSanitizedDataForEmail(email);
+        }
+        println("There are/is " + loopHelper.getCount() + " email(s).");
+        println("There are/is " + numberOfAffectedEmails + " affected email(s).");
+        println((isPreview ? 0 : numberOfUpdatedEmails) + " email(s) are/is successfully updated.");
```

Comment:
But in preview mode, numberofUpdatedEmails is always equal to numberOfAffectedEmail since the only line that can cause an Exception and cause numberOfUpdatedEmails not to be incremented is never ran. So it does not really help to give the approximate success rate.